### ISSUE TITLE
Param keys need to be strings, not symbols

### ISSUE THREAD

**Issue by luigi** [dbalatero/monster_mash#4]
The GoogleJSON example in the README doesn't work because Typhoeus expects the keys passed into params to be strings, not symbols. It would be nice if the symbols could be to_s'ed into strings before being passed to Typhoeus.

```
  >> GoogleJson.search("luigi montanez")
  NoMethodError: undefined method `<=>' for :q:Symbol
          from /Users/luigi/.gem/ruby/1.8/gems/typhoeus-0.1.24/lib/typhoeus/request.rb:45:in `sort'
          from /Users/luigi/.gem/ruby/1.8/gems/typhoeus-0.1.24/lib/typhoeus/request.rb:45:in `params_string'
          from /Users/luigi/.gem/ruby/1.8/gems/typhoeus-0.1.24/lib/typhoeus/request.rb:22:in `initialize'
          from /Users/luigi/.gem/ruby/1.8/gems/typhoeus-0.1.24/lib/typhoeus/request.rb:98:in `new'
          from /Users/luigi/.gem/ruby/1.8/gems/typhoeus-0.1.24/lib/typhoeus/request.rb:98:in `run'
          from /Users/luigi/.gem/ruby/1.8/gems/monster_mash-0.1.0/lib/monster_mash/request.rb:25:in `run_serial_request'
```

Gem versions installed:

typhoeus: 0.1.24
monster_mash: 0.1.0

**Comment by dbalatero** [comment]
Ah, good point! Nice catch. This was a problem I had with Typhoeus as well. Perhaps it would be best to fix it at the source in Typhoeus?

**Comment by dbalatero** [comment]
Updated the README to not confuse anymore, and I'll switch it back once I fix the params issue in Typhoeus.

**Comment by luigi** [comment]
Cool, thanks for the quick response.

**Comment by dbalatero** [comment]
This should now be fixed in Typhoeus 0.1.25.
